packages/fluentui/react-northstar/src/themes/teams/components/Chat/chatMessageStylesComfy.ts

```diff
@@ -0,0 +1,84 @@
+import { ComponentSlotStylesPrepared, ICSSInJSStyle } from '@fluentui/styles';
```

Comment:
nit: would go more for `classic` or `standard` here. `comfy` doesn't really say much about what it should look like, and since `compact` is a variant of the current layout it's easy to compare new layouts to something

Reply:
"Comfy" and "Compact" are the names we are using in Teams, I was reusing that terminology.
![Density settings](https://user-images.githubusercontent.com/2564094/121960418-e9c86400-cd1a-11eb-8860-29f717ce3914.png)

Reply:
Added doc-comments to both files.